source/style-guide/release-notes-guidelines.txt

```diff
@@ -158,36 +159,40 @@ Use the following guideline when creating items in each section:
      - Use sentences to describe the new feature or enhancement.
        Provide as needed, and provide a link to any section in the
        documentation that describes that feature.
-     - IPv6 support has been added. When you create a server, an IPv4
-       address and an IPv6 address block are assigned. The IPv4 address and
-       IPv6 address block are used for the public interface (Internet). The
-       IPv4 address is also used for the private interface.
+     - Supports IPv6. When you create a server, you can specify 
+       both IPv4 and IPv6 addresses.
 
    * - Resolved issues
      - Provide an initial phrase that describes the issue that was
-       fixed. Start the phrase with a past-tense verb. If necessary,
-       include sentences to further explain the fix. If you list only
+       fixed. Start the phrase with a verb. :ref:`Use present tense 
+       <use-present-tense>` or past tense consistently throughout the 
+       release notes. If necessary, include sentences to further 
+       explain the fix and the previous behavior. If you list only 
        phrases, do not use ending punctuation.
-     - Added support for Debian 8 (Jessie)
+     - - Supports Debian 8 (Jessie).
 
-       Trimmed the Monitoring ID and Monitoring Token configuration
-       variables to ensure correctness
+       - Trims the Monitoring ID and Monitoring Token configuration
+         variables to ensure correctness.
 
-       Fixed the Xen Server 6 package repo
+       - Fixes the Xen Server 6 package repository to resolve an issue 
+         that caused inconsistencies and errors with package 
+         management.
 
    * - Known issues
      - Use sentences to describe the issue. If a workaround is
        available, explain it.
-     - The create a service operation does not support Classless
-       Inter-Domain Routing (CIDR) for IP address restrictions.
+     - Results from the ``$indexStats`` operator exclude 
+       queries that use the ``$match`` or ``mapReduce`` functions.
 
    * - Documentation changes
      - Provide an initial phrase that describes the issue that was
-       fixed. Start the phrase with a past-tense verb. If necessary,
-       include sentences to further explain the fix. If you list only
-       phrases, do not use ending punctuation. Provide a link to the
-       relevant section in the documentation.
-     - Clarified the descriptions of the alert policies to include how
+       fixed. Start the phrase with a verb. :ref:`Use present tense 
+       <use-present-tense>` or past tense consistently throughout the 
+       release notes. If necessary, include sentences to further 
+       explain the fix. If you list only phrases, do not use ending 
+       punctuation. Provide a link to the relevant section in the 
+       documentation.
```

Comment:
Question: Could the example feature a link or placeholder for one since the recommendation is to provide a link? Or should this be optional?

"Clarifies... the alarm state. To learn more about ..., see \<link\>".